src/main/java/uk/gov/ons/census/casesvc/model/dto/RmUnInvalidateAddress.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.ons.census.casesvc.model.dto;
+
+import java.util.UUID;
+import lombok.Data;
+
+@Data
+public class RmUnInvalidateAddress {
```

Comment:
I mentioned in the toolbox PR too, I wonder if this should be `RmUninvalidateAddress`?